modin/core/storage_formats/pandas/parsers.py

```diff
@@ -311,10 +320,14 @@ def single_worker_read(cls, fname, *, reason: str, **kwargs):
 @doc(_doc_pandas_parser_class, data_type="CSV files")
 class PandasCSVParser(PandasParser):
     @staticmethod
-    @doc(_doc_parse_func, parameters=_doc_parse_parameters_common)
-    def parse(fname, **kwargs):
-        kwargs["callback"] = PandasCSVParser.read_callback
-        return PandasParser.generic_parse(fname, **kwargs)
+    @doc(_doc_parse_func, parameters=_doc_parse_parameters_common2)
+    def parse(fname, common_read_kwargs, **kwargs):
```

Comment:
is `common_read_kwargs` a dict here? Before adding this parameter we used `**kwargs`, which can be smth like `d = {"A": object_ref, "B": 1}`. Ray itself would deserialize `object_ref` in the remote task. If `common_read_kwargs` is a dict, Ray won't be able to deserialze `object_ref` containing in `common_read_kwargs`, which may slow down the flow.